### DISCUSSION TITLE
RuntimeError: The requested operation is not supported.

### DISCUSSION THREAD

**Discussion by Dharani-coding** [simpleble/simpleble#150] [Q&A] [answered]
I am able to execute write.py example of SimplepyBle. But running read.py example throws the below error.
![image](https://user-images.githubusercontent.com/46337466/208591592-48dea8f4-9b4a-4a14-8012-9b90edd3f548.png)
Please provide suggestions to resolve this issue.

**Comment by kdewald** [comment, marked as answer]
Hey @Dharani3110, thanks for reaching out.

That error message means that the characteristic does not list `read` as one of it's allowed operations (thus it's not supported). This does however raise the point that the error message is not clear enough, so I will fix that.
